pkg/artifactory/resource_artifactory_user.go

```diff
@@ -42,13 +54,12 @@ func resourceArtifactoryUser() *schema.Resource {
 			"admin": {
 				Type:        schema.TypeBool,
 				Optional:    true,
-				Computed:    true,
 				Description: "When enabled, this user is an administrator with all the ensuing privileges.",
 			},
 			"profile_updatable": {
 				Type:     schema.TypeBool,
 				Optional: true,
-				Computed: true,
+				Default:  true,
 				Description: "When enabled, this user can update their profile details (except for the password. " +
 					"Only an administrator can update the password). There may be cases in which you want to leave " +
 					"this unset to prevent users from updating their profile. For example, a departmental user with " +
```

Comment:
Please verify the need for `Computed` for the rest of the optional attributes. According to the [API schema](https://www.jfrog.com/confluence/display/JFROG/Security+Configuration+JSON#SecurityConfigurationJSON-application/vnd.org.jfrog.artifactory.security.User+json), they shouldn't be computed at all.

Reply:
Will check for other attributes in this resource. 
I thought, this fix to be point fix for this specific attribute only.

Reply:
I am always leary of expanding scope.

Reply:
1. Using `Computed` on those attributes are just wrong. I'm surprised we don't have reports of them not working correctly.
2. There are only 2 others attributes that have `Computed` set, so I think it's reasonable for us to fix them now. If there are more (says 5 or 10, spanning multiple resources) then I'd agree we shouldn't fix.